docs/pricing.md

```diff
@@ -118,3 +92,33 @@ task:
     task:
       use_compute_credits: $CIRRUS_BRANCH == 'master'
     ```
+
+## Compute Services
+
+Configure and connect one or several [compute services](guide/supported-computing-services.md) to Cirrus CI and [pay $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup) 
+for orchestrating CI workloads on these compute services. 
+
+**Pros** of this approach:
+
+* Full control of underlying infrastructure. Any type of VMs and containers with any amount of CPUs and memory.
+* Most secure. Setup any firewall and access rules.
+* Pay for CI within your existing cloud and GitHub bills. 
+  
+**Cons** of this approach:
+
+* Need to configure and connect one or several [compute services](guide/supported-computing-services.md). Might be
+  nonintuitive for cases like Anka Build Cloud for macOS.
+* Might not be worth the effort for a small team.
+* Need to buy $10/seat/month plan.
+
+!!! info "What is a seat?"
+
+    Seat is simply a GitHub user that initiates CI builds by pushing commints and/or creating pull requests in a **private** repository. 
+    It can be a real person or a bot. If you are using [Cron Builds](guide/writing-tasks.md#cron-builds) or creating builds thourgh [Cirrus API](api.md)
```

Comment:
```suggestion
    It can be a real person or a bot. If you are using [Cron Builds](guide/writing-tasks.md#cron-builds) or creating builds through [Cirrus's API](api.md)
```